abmodel/__init__.py

```diff
@@ -12,6 +12,20 @@
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
+#
+#This package is authored by:
+#Camilo Hincapié (https://www.linkedin.com/in/camilo-hincapie-gutierrez/) (main author)
+#Ian Mejía (https://github.com/IanMejia)
+#Emil Rueda (https://www.linkedin.com/in/emil-rueda-424012207/)
+#Nicole Rivera (https://github.com/nicolerivera1)
+#Carolina Rojas Duque (https://github.com/carolinarojasd)
+#and the conceptual contributions about epidemiology of
+#Lina Marcela Ruiz Galvis (mailto:lina.ruiz2@udea.edu.co).
+#
+#Other remarkably contributors to this work were
+#Alejandro Campillo (https://www.linkedin.com/in/alucardcampillo/)
+#Daniel Alfonso Montoya (https://www.linkedin.com/in/daniel-montoya-ds/).
```

Comment:
Delete this from all files, except from README.

Reply:
#and the conceptual contributions about epidemiology of
#Lina Marcela Ruiz Galvis (mailto:lina.ruiz2@udea.edu.co).

Remove also this from all files.